README.md

```diff
@@ -232,6 +232,17 @@ This collection of rules can be extended with custom rules using the `--append-r
 
 More information describing how rules are set up and an overview of all the Rules that are applied by this linter are documented [here](docs/rules.md).
 
+## Custom Rules
+
+The linter now supports the creation of custom one-line rules which compare any resource with a property using pre-defined operators. These custom rules take the following format:
+```
+<Resource Type> <Property[*]> <Operator> <Value> [Error Level] [Custom Error Message]
+```
+
+To use custom rules you will need to include `-z custom_rules.txt` or `--custom-rules custom_rules.txt`
```

Comment:
When I read a README I prefer to see quick examples of how it works instead of going straight into details. Consider writing something like this instead

```shell
$ cat custom_rules.txt
AWS::EC2::Instance InstanceSize NOT_EQUALS "m4.16xlarge" WARN "This is an expensive instance type, don't use it"

$ cat mqtemplate.yml
---
AWSTemplateFormatVersion: "2010-09-09"
Resources:
        myInstance:
                Type: AWS::EC2::Instance
                Properties:
                        InstanceType: m4.16xlarge
                        ImageId: ami-asdfef
$ cat .cfnlintrc
templates:
- mqtemplate.yml
include_checks:
- I
custom_rules:
- custom_rules.txt
$ cfn-lint
W0001  This is an expensive instance type, don't use it

mqtemplate.yml:6:17
```

and then you can link to the detailed documentation.

BTW, why is there an empty line between the message and the line location? The other rules don't show this

```
$ cfn-lint
E2522 At least one of [ImageId, LaunchTemplate] should be specified for Resources/myInstance/Properties
mqtemplate.yml:6:17

W0001  This is an expensive instance type, don't use it

mqtemplate.yml:6:17

```

Reply:
> BTW, why is there an empty line between the message and the line location? The other rules don't show this
> 
> ```
> $ cfn-lint
> E2522 At least one of [ImageId, LaunchTemplate] should be specified for Resources/myInstance/Properties
> mqtemplate.yml:6:17
> 
> W0001  This is an expensive instance type, don't use it
> 
> mqtemplate.yml:6:17
> ```

Oh that's hilarious - It's reading the newline at the end of your the rule as the first character of the "custom error message", but the detection for custom error message doesn't start unless there's alphanumeric characters detected after a space.

Will fix that, and redo the docs to include a simple go-to example on the initial front-page readme